official/utils/logging/hooks_helper.py

```diff
@@ -40,7 +40,7 @@ def get_train_hooks(name_list, **kwargs):
     name_list: a list of strings to name desired hook classes. Allowed:
       LoggingTensorHook, ProfilerHook, ExamplesPerSecondHook, which are defined
       as keys in HOOKS
-    kwargs: a dictionary of arguments to the hooks.
+    **kwargs: a dictionary of arguments to the hooks.
```

Comment:
This seems silly in the Args, but glint complains otherwise.

Reply:
Yeah, I got the same warning when glinting it . Will follow this style later. LGTM, thanks.